src/Sinch/Core/Http.cs

```diff
@@ -80,23 +86,87 @@ internal class Http : IHttp
                 $"sinch-sdk/{sdkVersion} (csharp/{RuntimeInformation.FrameworkDescription};;)";
         }
 
+        public Task<TResponse> SendMultipart<TRequest, TResponse>(Uri uri, TRequest request, Stream stream,
+            string fileName, CancellationToken cancellationToken = default)
+        {
+            var content = new MultipartFormDataContent();
```

Comment:
I think this part deserves to be put in a dedicated function such as `ConvertToMultipartFormDataContent` and be unit tested